plugins/coreboot/fu-plugin-coreboot.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2019 9elements Agency GmbH <patrick.rudolph@9elements.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#ifndef __packed
+#if defined(__WIN32) || defined(__WIN64)
+#define __packed __attribute__((gcc_struct, packed))
```

Comment:
Did you port all of fwupd to Windows? 😜
Probably don't need these ifdefs.

Reply:
removed